src/commands/usages/types.ts

```diff
@@ -0,0 +1,9 @@
+export type File = {
```

Comment:
this file was replaced by `types.d.ts` on the `hackweek` branch so that it doesn't return that error when running the `usages` command